code/modules/mob/mob.dm

```diff
@@ -1287,6 +1287,24 @@ var/list/slot_equipment_priority = list( \
 			var/mob/living/carbon/human/H = M
 			H.handle_regular_hud_updates()
 
+/mob/verb/rotate_chair() //To allow for actually sensible chair selection
```

Comment:
Is `for(var/[type] in [turf])` the fastest way to see if there's something on a turf? I know `locate()` is awful, but is it faster to just iterate through everything and check its type due to BYOND shittery? Because I kind of don't want to make mobs lag even worse than they already do

Or I guess I could just only make it do the check if the mob has a ckey, which would be fine in almost all cases.